applications/leaderboard/Leaderboard/Leaderboard.csproj

```diff
@@ -1,14 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Let's consider updating everything to 3.1 *after* this PR is done. I think it's been out for long enough that we can assume people can use it, at least for samples.